src/main/java/appeng/container/implementations/ContainerSpatialIOPort.java

```diff
@@ -37,7 +37,7 @@
 public class ContainerSpatialIOPort extends AEBaseContainer
 {
 
-	final TileSpatialIOPort spatialIOPort;
+	private final TileSpatialIOPort spatialIOPort;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "spatialIOPort" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)